src/05-invoicing/product-categories.apib

```diff
@@ -0,0 +1,21 @@
+### productCategories.list [GET /productCategories.list]
+
+Get a list of product categories.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + department_id: `080aac72-ff1a-4627-bfe3-146b6eee979c` (string, optional) - The ID of the department you wish to filter on.
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `2aa4a6a9-9ce8-4851-a9b3-26aea2ea14c4` (string)
+                + name: 'Asian Flowers' (string)
+                + ledgers (array)
+                    + (object)
+                        + department_id: `2aa4a6a9-9ce8-4851-a9b3-26aea2ea14c6` (string)
```

Comment:
Should this be done like this @stivni ?
Shouldn't we use the same id format everywhere??

Reply:
Yes; https://github.com/teamleadercrm/api/blob/master/GUIDELINES.md#returning-relationships

Reply:
Good catch.

Reply:
thanks :D